### ISSUE TITLE
Import button doesn't seem to do anything anymore

### ISSUE THREAD

**Issue by kiaric** [spetzel2020/easy-exports#11]
I'm on the latest Foundry version (v9, build 255) and it seems like the import function doesn't work at all anymore. I've tried a full export and a smaller single folder export, the exported files themselves seem fine, but when I go to import them, nothing happens. I've waited about 5-10 minutes on some occasions just to see if I'd get any notification that the import worked, but nothing. 

Tried on two separate worlds, tried with all other modules off. Nothing showing up in the compendiums tab.

**Comment by spetzel2020** [comment]
Thanks for the report; I'll take a look.

**Comment by spetzel2020** [comment]
Reproduced with Foundry 9.255. The problem is occurring because the export file is incorrect; instead of the Entity name of the export, it has `undefined`. And that is occurring because the export is continuing to use `.entity` which was deprecated and has been removed (but apparently not immediately in Foundry v9).

WORKAROUND: If you really need to import the export file you have created, you can change `undefined` to the correct entity in the `.json` export file. 

Example: bad file:
```
{"undefined":[{
  "_id": "00pPIPbCQHVCAGsZ",
  "name": "23.1.10.02 10b. Shield Trap",
...
```

Fixed file:
```
{"JournalEntry":[{
  "_id": "00pPIPbCQHVCAGsZ",
  "name": "23.1.10.02 10b. Shield Trap",
...
```

**Comment by spetzel2020** [comment]
The export is failing here in `EasyExport.exportTree()#line253-254`:
```
    //Prepend and append [] for array re-import
    const entity = this.documents[0]?.entity;
```
should be:
```
 const entity = this.documents[0]?.documentName;
```

**Comment by spetzel2020** [comment]
Fixed (plus other deprecation warning) in 0.6.5

**Comment by spetzel2020** [comment]
Released in 0.6.5
